Code/Editor/TrackView/AtomOutputFrameCapture.cpp

```diff
@@ -47,18 +48,39 @@ namespace TrackView
         AZ::Name viewName = AZ::Name("MainCamera");
         m_view = AZ::RPI::View::CreateView(viewName, AZ::RPI::View::UsageCamera);
         m_renderPipeline->SetDefaultView(m_view);
+        m_targetView = scene.GetDefaultRenderPipeline()->GetDefaultView();
     }
 
     void AtomOutputFrameCapture::DestroyPipeline(AZ::RPI::Scene& scene)
     {
+        AZ::Render::PostProcessFeatureProcessor* fp = scene.GetFeatureProcessor<AZ::Render::PostProcessFeatureProcessor>();
+        if (fp)
+        {
+            fp->RemoveViewAlias(m_view);
+        }
         scene.RemoveRenderPipeline(m_renderPipeline->GetId());
         m_passHierarchy.clear();
         m_renderPipeline.reset();
         m_view.reset();
+        m_targetView.reset();
     }
 
-    void AtomOutputFrameCapture::UpdateView(const AZ::Matrix3x4& cameraTransform, const AZ::Matrix4x4& cameraProjection)
+    void AtomOutputFrameCapture::UpdateView(const AZ::Matrix3x4& cameraTransform, const AZ::Matrix4x4& cameraProjection, const AZ::RPI::ViewPtr targetView)
     {
+        if (targetView && targetView != m_targetView)
+        {
+            AZ::RPI::Scene* scene = SceneFromGameEntityContext();
+            if (scene)
```

Comment:
nit/suggestion:
```
if (AZ::RPI::Scene* scene = SceneFromGameEntityContext())
```
or even...
```
if (AZ::RPI::Scene* scene = SceneFromGameEntityContext(); scene != nullptr)
// or
if (AZ::RPI::Scene* scene = SceneFromGameEntityContext(); scene)
```
If you'd prefer to be more explicit, the reduced scope is always useful and guards against misuse later in the function by accident